### ISSUE TITLE
list_up and down doesn't work properly 

### ISSUE THREAD

**Issue by Moo2017** [lvgl/lvgl#197]
Hi All,

I want to use List obj, I wrote the following code, I am using the hardware keyboard. Once I move down I can not see the last item of the list , if I change the list position (    lv_obj_set_pos(list2, 10, 60);) I am not able to see the two last items and so on. I would appreciate if someone help me regarding this issue.

void lv_test_list_1(void)
{

    list2 = lv_list_create(lv_scr_act(), NULL);
    lv_obj_set_pos(list2, 10, 10);

    lv_list_add(list2, SYMBOL_FILE, "File", NULL);
    lv_list_add(list2, SYMBOL_DIRECTORY, "Directory", NULL);
    lv_list_add(list2, SYMBOL_UP, "Up", NULL);
    lv_list_add(list2, SYMBOL_DOWN, "Down", NULL);
    lv_list_add(list2, NULL, "No icon", NULL);
    lv_list_add(list2, SYMBOL_CLOSE, "", NULL);
    lv_list_add(list2, SYMBOL_UP, "Up", NULL);
    lv_list_add(list2, SYMBOL_DOWN, "1", NULL);
    lv_list_add(list2, SYMBOL_DOWN, "2", NULL);
    lv_list_add(list2, SYMBOL_DOWN, "3", NULL);
    lv_list_add(list2, SYMBOL_DOWN, "4", NULL);

    lv_obj_set_width(list2, LV_HOR_RES-80);

              g1 = lv_group_create();
              lv_indev_set_group(kb_indev, g1);

    /*Add list up/down buttons*/
    lv_obj_t *btn_up = lv_btn_create(lv_scr_act(), NULL);
  //  lv_obj_align(btn_up, NULL, LV_ALIGN_IN_BOTTOM_LEFT, 0, 10);
    lv_obj_set_width(btn_up, 50);
    lv_obj_set_pos(btn_up,LV_HOR_RES-60,LV_VER_RES-50);
    lv_btn_set_action(btn_up, LV_BTN_ACTION_CLICK, list_move);
    lv_obj_set_free_num(btn_up, 0);
    lv_obj_t *label = lv_label_create(btn_up, NULL);
    lv_label_set_text(label, SYMBOL_UP);

    lv_obj_t *btn_down = lv_btn_create(lv_scr_act(), btn_up);
      lv_obj_set_pos(btn_down,LV_HOR_RES-60,LV_VER_RES-120);
    lv_obj_set_free_num(btn_down, 1);
    label = lv_label_create(btn_down, NULL);
    lv_label_set_text(label, SYMBOL_DOWN);
//    lv_list_up(list3);
  //  lv_list_up(list4);
    lv_group_add_obj(g1, btn_up);
    lv_group_add_obj(g1, btn_down);
//    lv_group_add_obj(g1, list2);

}

/**********************
 *   STATIC FUNCTIONS
 **********************/

static lv_res_t list_move(lv_obj_t *btn)
{
    if(lv_obj_get_free_num(btn) == 0) {
        lv_list_up(list2);

    } else {
        lv_list_down(list2);
    }
    return LV_RES_OK;
}

**Comment by kisvegabor** [comment]
Hi,

It seems you are using this concept a little bit differently as it was intended.
1. You don't have to use `lv_list_up/down` in this case.
2. You just need to focus on the list with the group and send LV_GROUP_KEY__UP/DOWN to it.

I added these 2 lines at the end if your `lv_test_list_1` function and it now it worked well:
```c
lv_group_add_obj(g1, list2);
lv_group_focus_obj(list2);
```

**Comment by Moo2017** [comment]
In fact I added the list to the group , same as what you did, but I had problem in moving up and down when list item are more then screen height , then I checked the existing example from the lv_boards and try test that to make sure I am doing something wrong or the issue exists. 
The problem was because I haven't set the list height manually. I set and now list move along with the cursor. 
Since I am actively use this library in one of our company project I'd like to know how can I get more Support from you? I found that you have some services but I don't know how can I get a quote, I sent an email but I didn't get any response. I'd appreciate if you let me know..
